1.4.230 build fails

### Description

I found myself trying to rebuild mumble 1.4.230 after another update with custom settings.
But I failed to do that, because of error
```
CMake Error at overlay_gl/CMakeLists.txt:37 (CHECK_INCLUDE_FILE):
  Unknown CMake command "CHECK_INCLUDE_FILE".
```
Is it arch problem, or https://cmake.org/cmake/help/latest/module/CheckIncludeFile.html macros should be included in upstream?

### Steps to reproduce

1. Get archlinux. For example I test it with
   `docker run -it --rm archlinux bash`
2. Prepare building
   1. `pacman -Sy asp base-devel`
   2. `useradd -m user`
   3. `echo 'user ALL=(ALL:ALL) NOPASSWD: ALL' >> /etc/sudoers`
   5. `sudo -i -u user`
   6. `asp checkout mumble`
   7. `cd mumble/repos/community-x86_64`
   8. `gpg --recv-keys 25F63C66245DFC60`
3. `makepkg -s`


### Mumble version

1.4.230

### Mumble component

Client

### OS

Linux

### Reproducible?

Yes

### Additional information

I patched this problem with the following patch
```diff
--- a/overlay_gl/CMakeLists.txt
+++ b/overlay_gl/CMakeLists.txt
@@ -34,6 +34,7 @@ if(NOT APPLE)
 		# Just check for this header file while using a 32bit target as a really small and incomplete check whether g++-multilib seems to be
 		# installed. If we don't find it, we can assume it's not there but if we do find it, we still don't know. Thus we still print the
 		# message about the 32bit target potentially failing due to missing g++-multilib.
+		INCLUDE(CheckIncludeFile)
 		CHECK_INCLUDE_FILE("sys/cdefs.h" FOUND_CDEFS "-m32")
 		if(NOT FOUND_CDEFS)
 			message(FATAL_ERROR "Can't find the 32bit version of sys/cdefs.h - did you install g++-multilib?")
```

### Relevant log output

_No response_

### Screenshots

_No response_

--- a/overlay_gl/CMakeLists.txt
+++ b/overlay_gl/CMakeLists.txt
@@ -5,6 +5,8 @@
 
 # Overlay payload for UNIX-like systems.
 
+include(CheckIncludeFile)
+
 if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64" OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "amd64")
 	if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
 		option(overlay-xcompile "Build 32 bit overlay library, necessary for the overlay to work with 32 bit processes." ON)

